src/walletConnect/screens/ActionRequest.tsx

```diff
@@ -83,7 +83,7 @@ function ActionRequest({ navigation, route: { params: routeParams } }: Props) {
       : method === SupportedActions.personal_decrypt
       ? Buffer.from(params[1]).toString('hex')
       : method === SupportedActions.personal_sign
-      ? params[0]
+      ? Buffer.from(params[0].replace(/^0x/, ''), 'hex').toString()
```

Comment:
If the decoded message is an empty string (in case of invalid hex), this results in an empty string and causes the <Text> component to crash. Not sure if this is possible at all in practice since personal sign always sends a hex string, but should we still be defensive to it?

Reply:
ideally yes

Reply:
kk, any preference on what to show in that case, the raw hex? or just make it `null` so "Show Details" button won't show up? or something else?

Reply:
chatted offline and decided to go with "Invalid message" string.